build.gradle.kts

```diff
@@ -19,15 +19,27 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("org.springframework.boot:spring-boot-starter-validation")
     implementation("org.springframework.boot:spring-boot-starter-actuator")
+    implementation("junit:junit:4.13.1")
```

Comment:
This worries me. We are already using JUnit (5 I think) successfully - not sure what the impact is of adding JUnit 4 as well?

Reply:
removed